projects/exui-common-lib/src/lib/components/search-location/search-location.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="govuk-form-group">
+    <div class="location-picker-custom">
+        <label class="govuk-body" for="input-selected-location">
+            Search for a location by name
+        </label>
+        <br>
+        <div class="auto-complete-container">
+            <div>
+                <div class="autocomplete__wrapper">                
+                        <input 
+                            [formControl]="findLocationForm.controls.findLocationControl"
+                            [matAutocomplete]="auto"
+                            aria-expanded="false" 
+                            aria-owns="input-selected-location__listbox" 
+                            aria-autocomplete="both" 
+                            aria-describedby="input-selected-location__assistiveHint" 
```

Comment:
This will cause accessibility issue, aria-describedby needs to be binded with an element id, but found nothing on 'input-selected-location__assistiveHint'
see how to use it here:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute